config/initializers/features.rb

```diff
@@ -0,0 +1,38 @@
+# Initialize our store of feature flags using rollout
+module Alaveteli
+  module Features
+    # Rollout assumes you'll use Redis for storing key:value pairs that
+    # describe features and who can use them. For now we're storing this in an
+    # ActiveRecord model instead for simplicity, so we configure Rollout to
+    # use that instead.
+    @@rollout = Rollout.new(RolloutKeyValueStore.new)
```

Comment:
Not sure whether using a "class" variable has the same problems as when using them in a class. Probably want to  make this a singleton-style module/gem in lib anyway, e.g. https://github.com/mysociety/alaveteli/blob/develop/lib/alaveteli_geoip.rb